GameData/RealSolarSystem/RSSKopernicus.cfg

```diff
@@ -896,9 +896,9 @@
 		}
 		PQS
 		{
-			deactivateAltitude = 130000
-			fadeStart = 80000
-			fadeEnd = 120000
+			deactivateAltitude = 101000

```

Comment:
@stratochief66 Deactive altitude is also activate altitude. It should be well above the fadeEnd value. or you may get a transparent planet for a frame or two while PQS builds.